TreeTracker/Services/SelfieService.swift

```diff
@@ -60,4 +65,20 @@ class LocalSelfieService: SelfieService {
             completion(.failure(error))
         }
     }
+
+    func fetchSelfie(forPlanter planter: Planter, completion: (Result<Data, Error>) -> Void) {
```

Comment:
Im not sure we need this as we can already access from the planter object.

Maybe some extensions in Planter.swift similar to in TreeService.swift like...

```
var latestIdentification: PlanterIdentification? {
    let sortedPlanterIdentification = identification?.sorted(by: { (lhs, rhs) -> Bool in
        guard let lhs = lhs as? PlanterIdentification,
            let rhs = rhs as? PlanterIdentification,
            let lhsDate = lhs.createdAt,
            let rhsDate = rhs.createdAt else {
                return false
        }
        return lhsDate > rhsDate
    })
    return sortedPlanterIdentification?.first as? PlanterIdentification
}

var latestIdentificationPhoto: Data {
    return latestIdentification.localPhotoPath
}
```

Then we can do `planter.latestIdentificationPhoto` in the HomeViewModel

Reply:
hey @AlexCornforth i still think we need this because we need to someone fetch the selfie information to then pass it back to the view model. line 72 is using the proper `lastestIdentification` extension function

Reply:
Yeh I would agree with you 👍